app/uk/gov/hmrc/exports/models/declaration/ExportItem.scala

```diff
@@ -39,23 +42,26 @@ object AdditionalFiscalReferences {
 }
 
 case class ItemType(
-                     combinedNomenclatureCode: String,
-                     taricAdditionalCode: Seq[String],
-                     nationalAdditionalCode: Seq[String],
-                     descriptionOfGoods: String,
-                     cusCode: Option[String],
-                     unDangerousGoodsCode: Option[String],
-                     statisticalValue: String
+  combinedNomenclatureCode: String,
+  taricAdditionalCode: Seq[String],
+  nationalAdditionalCode: Seq[String],
+  descriptionOfGoods: String,
+  cusCode: Option[String],
+  unDangerousGoodsCode: Option[String],
+  statisticalValue: String
 )
 object ItemType {
   implicit val format: OFormat[ItemType] = Json.format[ItemType]
+
+  object IdentificationTypeCodes {
+    val CombinedNomenclatureCode = "TSP"
```

Comment:
Same here - this looks like Enum - maybe seald trait/abstract class with some case objects in companion would be enough

Reply:
this has been addressed